### PR TITLE
[IGNORE] README edits

### PR DESCRIPTION
**Pull request by crandmck**
- Change terminology to align with the rest of the docs: This repo is now referred to as the "Rust library" instead of the "Rust SDK". 
- Fixed header level for releases under "Release notes"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,29 +116,32 @@ Please note that there is no formal support for code from a nightly release, but
 
 This section gives a highlight of noteworthy changes.
 
-Refer to the [CHANGELOG](https://github.com/contentauth/c2pa-rs/blob/main/CHANGELOG.md) for detailed changes derived from git commit history.
+Refer to the [CHANGELOG](https://github.com/contentauth/c2pa-rs/blob/main/CHANGELOG.md) for detailed changes derived from Git commit history.
```
Comment (scouten-adobe) [inline README.md:119]: Common misconception: "Git" should be capitalized. Please refer to git in lowercase as is preferred usage for this tool.

**Review by scouten-adobe — CHANGES_REQUESTED** [review]
Mostly good, but I did request a change regarding git usage.

File: README.md (hunk 1 above)
Comment (crandmck) [reply README.md:119]: No, Git is a proper noun, and is thus capitalized.  However `git` the command is not. 

Supporting references:
- https://git-scm.com/ This is the website for the project and is thus definitive. (n.b. the home page has "Git" only as the first word in a sentence and so is not a helpful guide, but elsewhere on the site it's consistently capitalized).
- https://english.stackexchange.com/questions/611711/tech-related-should-i-capitalize-the-word-git-in-this-context-or-not
- - https://github.com/swcarpentry/git-novice/issues/505
- https://en.wikipedia.org/wiki/Git

**Review by gpeacock — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gpeacock) [reply README.md:119]: This sounds correct to me

**Review by scouten-adobe — APPROVED** [review]

**Pull request merged**